Explorer/Assets/DCL/InWorldCamera/CameraReelGallery/CameraReelGalleryController.cs

```diff
@@ -244,22 +282,47 @@ private async UniTask DeleteScreenshotsAsync(ReelToDeleteInfo reelToDeleteInfo,
             }
         }
 
-        public async UniTask ShowWalletGalleryAsync(string walletAddress, CancellationToken ct, CameraReelStorageStatus? storageStatus = null)
+        private void PrepareShowGallery(CancellationToken ct)
         {
+            view.loadingSpinner.SetActive(true);
             loadNextPageCts = loadNextPageCts.SafeRestartLinked(ct);
             setPublicCts = setPublicCts.SafeRestart();
             deleteScreenshotCts = deleteScreenshotCts.SafeRestart();
 
             view.scrollRect.verticalNormalizedPosition = 1f;
             previousY = 1f;
+        }
+
+        private void FinishShowGallery()
+        {
+            view.scrollRect.onValueChanged.AddListener(OnScrollRectValueChanged);
```

Comment:
are you removing this listener somewhere? cause I see you are calling FinishShowGallery in ShowPlaceGalleryAsync without removing any previous listener

Reply:
yes, it is called in `OnDisable`